README.rst

```diff
@@ -8,31 +8,45 @@ can create a fresh Django site project. It replaces the old "nensskel" tool.
 Using this cookiecutter template
 --------------------------------
 
-Install cookiecutter ("pip install cookiecutter").
+Install/upgrade cookiecutter and pipenv::
+
+  $ pip install cookiecutter pipenv --upgrade
+
 
 Run the following command and answer the questions::
 
   $ cookiecutter https://github.com/nens/cookiecutter-djangosite-template
 
+Decide whether your project is an application or a library (`read more <https://docs.pipenv.org/advanced/#pipfile-vs-setuppy>`_ ).
```

Comment:
Deze template is voor een Django site. Dus niet voor een library?

Reply:
Das waar, dat versimpelt de boel wel.